private-media-framework/draft-ietf-perc-private-media-framework.md

```diff
@@ -332,40 +334,42 @@ security contexts and two associated encryption keys: an "inner" key
 encryption of RTP media between endpoints and an "outer" key (HBH key)
 known only to media distributor and the adjacent endpoint)
 for the hop between an endpoint and a Media Distributor or between Media
-Distributor.  Reference the following figure.
+Distributor.  
 
 ~~~
 +-------------+                                +-------------+
 |             |################################|             |
-|    Media    |------------------------------->|    Media    |
-| Distributor |<-------------------------------| Distributor |
-|      X      |################################|      Y      |
-|             |          HBH Key (XY)          |             |
-+-------------+                                +-------------+
-   #  ^ |  #                                      #  ^ |  #
-   #  | |  #       HBH                  HBH       #  | |  #
-   #  | |  # <== Key(AX)              Key(YB) ==> #  | |  #
+|    Media    |-----------------------*------->|    Media    |
+| Distributor |<----------------------*--------| Distributor |
+|      X      |#####################*#|########|      Y      |
+|             |                     | |        |             |
++-------------+                     | |        +-------------+
+   #  ^ |  #          HBH Key (XY) -+ |           #  ^ |  #
+   #  | |  #                          |           #  | |  #
+   #  | |  #          E2E Key (AB) ---+           #  | |  #
```

Comment:
Can we split out Key (A) and Key (B) in each place?  I don't want the reader confused thinking this is a bidirectional key or key targeting a specific endpoint.